Docs/README.md

```diff
@@ -64,65 +72,101 @@ Splice reads:                                  4161455
 Reads mapped in proper pairs:                  21856264
 Proper-paired reads map to different chrom:    7648
 ```
+The MultiQC plot of this information can be a bit overwhelming at first but it displayes the above information simultaneously for all the samples in your report. With each sample represented as a dot. 
 ###infer_experiment.py
 
-This scripts guesses how the reads were stranded for strand specific RNA-seq data. Example output:
+This scripts guesses how the reads were stranded for strand specific RNA-seq data. 
+Example output from an unstranded(~50% sense/antisense) library of paired end data:
 
-```
-infer_experiment.py -r hg19.refseq.bed12 -i Pairend_nonStrandSpecific_36mer_Human_hg19.bam
+**From MultiQC report:**
+<img src= "images/infer_experiment.png">
 
-#Output:
+**From the infer_experiment.txt file:**
 
+```
 This is PairEnd Data
-Fraction of reads failed to determine: 0.0172
-Fraction of reads explained by "1++,1--,2+-,2-+": 0.4903
-Fraction of reads explained by "1+-,1-+,2++,2--": 0.4925
+Fraction of reads failed to determine: 0.0409
+Fraction of reads explained by "1++,1--,2+-,2-+": 0.4839
+Fraction of reads explained by "1+-,1-+,2++,2--": 0.4752
 ```
 
+
+
 ###junction_saturation.py
 This scripts aims to determine if the used sequencing depth is enough to perform  alternative splicing analysis on the data. 
 
-![Junction Saturation](images/junction_saturation.png)
+![Junction Saturation](images/Junction_saturation.png)
 
-In the above example output the red line (known junction) has reached a plateau. Nearly all known junctions in this data set have already been detected, and deeper sequencing will thus not help in detecting more known junctions. The green line (novel junction) has not plateaued and thus further sequencing will likely aid in junction saturation.
-In summary, if the lines are flattening out then you have sequenced deeply enough. 
-###RPKM saturation
+
+A line that has plateaued indicates that all junctions have been detected and that further secuencing will not yield more results. Insteads the same fragment would be sequenced again and again. None of the lines in the example above have plataued and thus these samples could reveal more alternative splicing information if they were sequenced deeper. 
```

Comment:
_indicates that all junctions have been detected_ - this is not true. It means that the sample has been sequenced to saturation. It could be a very tiny / low complexity sample with hardly any junctions represented in the library (different to detecting _all_ junctions).

> The x-axis shows each library at different down-sampling percentages. If the library has been sequenced to saturation, the line will plateau and increasing sequencing depth will not yield any additional splicing information.